vue/components/ui/molecules/input-currency/input-currency.vue

```diff
@@ -28,7 +28,7 @@ export const InputCurrency = {
             default: () => ({})
         },
         value: {
-            type: String,
+            type: Number,

```

Comment:
Maybe we should accept both `String` and `Number`. Then, we can convert it to a `Number` every time we copy this to `valueData`.